### PR TITLE
Form for adoption

### PR DESCRIPTION
**Pull request by MtnGreen**
This isn't final, but we need everyone to have the component, so they can all test in their own browsers.

### CODE REVIEW

**Comment by MtnGreen** [comment]
Fixed AdoptionForm and NewPetForm. Ready for review.

File: src/components/AdoptionForm.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const handleAppSubmit = event => {
+    event.preventDefault()
+    let payload = {
+      name: newApplication.name,
+      phoneNumber: newApplication.phoneNumber,
+      email: newApplication.email,
+      homeStatus: newApplication.homeStatus
+    }
+
+
```
Comment (DanAlberts) [inline src/components/AdoptionForm.js]: One too many blank lines here

File: src/components/AdoptionForm.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        body: JSON.stringify(payload)
+      })
+        .then((result) => {
+          if(result.ok){
+          setAdoptAppStatus("Your request is in process")}
+        })
+        .catch(error => {
+          console.log(error)
+        })
+
```
Comment (DanAlberts) [inline src/components/AdoptionForm.js]: Unnecessary blank line here

File: src/components/AdoptionForm.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        <label htmlFor="email">Email Address:
+        <input type="text" name="email" id="email" onChange={handleAppChange} value={newApplication.email} />
+        </label>
+        <label htmlFor="homeStatus">Select Home Status</label>
+        <select name="homeStatus" id="homeStatus" onChange={handleAppChange} value={newApplication.homeStatus}>
+          <option value="default" disabled hidden>Select Home Status</option>
+          <option value="own">Own</option>
+          <option value="rent">Rent</option>
+        </select>
+        <div>
```
Comment (DanAlberts) [inline src/components/AdoptionForm.js:74]: Looks like the form went back to not being indented correctly here.

**Review by DanAlberts — CHANGES_REQUESTED** [review]

File: src/components/AdoptionForm.js (hunk 1 above)
Comment (MtnGreen) [reply src/components/AdoptionForm.js]: Fixed!

File: src/components/AdoptionForm.js (hunk 2 above)
Comment (MtnGreen) [reply src/components/AdoptionForm.js]: Fixed!

File: src/components/AdoptionForm.js (hunk 3 above)
Comment (MtnGreen) [reply src/components/AdoptionForm.js:74]: Fixed!

**Comment by MtnGreen** [comment]
All fixed. Thanks!

**Review by DanAlberts — APPROVED** [review]

**Pull request merged**